.github/workflows/pkgci_regression_test.yml

```diff
@@ -208,6 +208,19 @@ jobs:
         env:
           ROCM_CHIP: ${{ matrix.rocm-chip }}
 
+      - name: "Running SD3 special model tests"
+        if: "!cancelled()"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest ./experimental/regression_suite/shark-test-suite-models/sd3 \
```

Comment:
Did you mean to run the llama tests here? This looks copy/pasted from the block just above.

Reply:
uhhhh weird thats not what it says locally. Maybe I didn't save the file before git add. thanks